documentation/src/main/asciidoc/topics/con_shared_cache_stores.adoc

```diff
@@ -0,0 +1,37 @@
+[id='shared-cache-stores_{context}']
+= Shared cache stores
+
+{brandname} cache stores can be local to a given node or shared across all nodes in the cluster.
+
+Configuring cache stores as shared, `shared`, or local only, `shared(false)`,  impacts the rest of your persistent storage configuration.
```

Comment:
shared, `shared`, or local only, `shared(false)`

Did you meant to put shared in there twice like that? Should it be one for false and the other for true.